### PR TITLE
LJE-50 - LJE-61  & LJE-62

### PR DESCRIPTION
**Pull request by zhuTRN**
Hey Team, I have completed a functional navbar component that contains the e-commerce shop title with the logo. The navbar has 4 buttons to go to the homepage, product listing page, sign in and sign up page. These pages also have the navbar at the top to go to different directories. Please take a look and approve my pull request, Thanks!

### CODE REVIEW

File: packages/frontend/src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+`
+
+const Category = styled.a`
+    
+`
+// const Logo = styled.i`
+//     margin-left: 0px;
+// `
+
+class Navbar extends Component {
```
Comment (fahadahmed) [inline packages/frontend/src/components/Navbar/Navbar.tsx:48]: Please convert this to a functional component. Class components are an old way to write React components which will become redundant soon.

**Review by fahadahmed — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (zhuTRN) [reply packages/frontend/src/components/Navbar/Navbar.tsx]: Hey Fahad, thanks for the review, I have now converted Navbar into a functional component, please let me know if there is any problem.

**Comment by zhuTRN** [comment]
Hey Fahad, thanks for the review, I have now converted Navbar into a functional component, please let me know if there is any problem.

File: packages/frontend/src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+//                 <div className='nav-container'>
+//                     <Link to='/' className='navbar-logo'>
+//                         Navbar it is
+//                     </Link>
+//                     <div className="menu-icon"></div>
+//                 </div>
+//             </nav>
+//         </div>
+//     )
+// }
```
Comment (fahadahmed) [inline packages/frontend/src/components/Navbar/Navbar.tsx:101]: Hi Brian. Can you also remove the commented out code?

File: packages/frontend/src/components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+const Navbar = () => {
+    // const [clicked, setclicked] = useState(false);
+
+    // const handleClick = () => {
+    //     setState({ clicked: !state.clicked })
+    // }
+
+    // const NavMenu = () => {
+    //     return this.state.clicked ? NavMenuActive : NavMenuInactive
+    // }
```
Comment (fahadahmed) [inline packages/frontend/src/components/Navbar/Navbar.tsx:57]: Same here. please remove the commented out code.

File: packages/frontend/src/components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    justify-content: end;
+    margin-right: 2rem;
+`
+
+const Category = styled.p`
+    color: white;
+
+// const Logo = styled.i`
+//     margin-left: 0px;
+// `
```
Comment (fahadahmed) [inline packages/frontend/src/components/Navbar/Navbar.tsx:46]: please remove the commented out code.

File: packages/frontend/src/components/Navbar/MenuItems.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        title: 'Products',
+        url: '#',
+        cName: '/product-list'
+    },
+    // {
+    //     title: 'Contact Us',
+    //     url: '#',
+    //     cName: 'nav-links'
+    // },
```
Comment (fahadahmed) [inline packages/frontend/src/components/Navbar/MenuItems.tsx:16]: please remove the commented out code.

File: packages/frontend/src/pages/Home/Home.tsx [hunk 6]
```diff
@@ -1,13 +1,19 @@
...
+import { Global, css } from '@emotion/react'
 import { cp } from "fs";
+import Navbar from '../.././components/Navbar'
 
 const Container = styled.div`
     display: grid;
-    justify-content: center;
-    align-items: center;
+    /* justify-content: center; */
+    /* align-items: center; */
```
Comment (fahadahmed) [inline packages/frontend/src/pages/Home/Home.tsx:11]: please remove the commented out code here as well.

**Review by fahadahmed — CHANGES_REQUESTED** [review]

**Comment by fahadahmed** [comment]
Hi @zhuTRN , thanks for completing the conversion of the navbar component to a functional one. Just a minor change. Can you please remove any commented out code in your PR. We would like to keep the code as clean as possible.

**Review by fahadahmed — APPROVED** [review]

**Pull request merged**